src/Garn.hs

```diff
@@ -48,7 +48,7 @@ runWith env (WithGarnTsOpts garnConfig opts) = do
   case opts of
     Gen -> pure ()
     Run (CommandOptions {..}) argv -> do
-      callProcess "nix" $ ["run"] <> nixArgs <> [".#" <> asNixFacing target, "--"] <> argv
+      rawSystem "nix" (["run"] <> nixArgs <> [".#" <> asNixFacing target, "--"] <> argv) >>= throwIO
```

Comment:
```suggestion
      exitCode <- rawSystem "nix" (["run"] <> nixArgs <> [".#" <> asNixFacing target, "--"] <> argv)
      exitWith exitCode
```

I think this makes it a bit clearer that this is not an exceptional case where we throw, but instead just exiting with the proper exit code.

(`rawSystem` seems fine to use.)

Reply:
This certainly works for me.